cpp/log/cert.cc

```diff
@@ -236,6 +236,31 @@ util::Status Cert::LoadFromDerBio(BIO* bio_in) {
 }
 
 
+string Cert::PrintVersion() const {
+  if (!IsLoaded()) {
+    LOG(ERROR) << "Cert not loaded";
+    return string();
+  }
+
+  const long version(X509_get_version(x509_.get()));
+  return std::to_string(1 + version);
```

Comment:
There's a `using std::to_string` at the top, so you can drop the `std::` qualifier.